pkg/sqlite/load.go

```diff
@@ -627,6 +629,7 @@ func (s *sqlLoader) getCSVNames(tx *sql.Tx, packageName string) ([]string, error
 	if err != nil {
 		return nil, err
 	}
+	defer rows.Close()
```

Comment:
`rows.Close()` is called a few lines below (not deferred to capture the error)

Reply:
fixed this up